apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/nodes/StepBlockNode/libs/getPriorityImage/getPriorityImage.ts

```diff
@@ -0,0 +1,22 @@
+import { TreeBlock } from '@core/journeys/ui/block'
+import {
+  BlockFields,
+  BlockFields_ImageBlock as ImageBlock
+} from '@core/journeys/ui/block/__generated__/BlockFields'
+
+export function getPriorityImage(
+  card: Array<TreeBlock<BlockFields>>
```

Comment:
```suggestion
  cardChildrenBlocks: Array<TreeBlock<BlockFields>>
```

Since when you consume this function, you aren't passing in the card but the children of the card